src/calendar/header/index.tsx

```diff
@@ -279,7 +280,7 @@ const CalendarHeader = forwardRef((props: CalendarHeaderProps, ref) => {
       testID={testID}
       style={propsStyle}
       accessible
-      accessibilityRole={'adjustable'}
+      accessibilityRole={accessibilityRole}
```

Comment:
You're removing the existing value without even set it as the default...

Reply:
added fallback value to "adjustable"